raiden/transfer/views.py

```diff
@@ -133,6 +133,24 @@ def total_deposit_by_token_network(
     return total_deposit
 
 
+def get_token_network_identifier_by_token_address(
+        node_state: NodeState,
+        payment_network_id: typing.Address,
+        token_address: typing.Address,
+) -> typing.Address:
+    token_network = get_token_network_by_token_address(
+        node_state,
+        payment_network_id,
+        token_address,
+    )
+
+    token_network_id = None
```

Comment:
nitpick: Wouldn't these 6 lines be nicer if reduced to a ternary operator?:

`return token_network.address if token_network is not None else None`